client/my-sites/marketplace/components/billing-interval-switcher/index.tsx

```diff
@@ -48,6 +65,19 @@ const BillingIntervalSwitcher: React.FunctionComponent< Props > = ( {
 						onClick={ () => onChange( IntervalLength.ANNUALLY ) }
 					>
 						{ annualLabel }
+						{ plugin && (
+							<PluginAnnualSaving plugin={ plugin }>
+								{ ( annualSaving: { saving: never } ) =>
```

Comment:
The type of this argument is confusing to me. `never` means that it will never be set (I think), but then you're using it below. Was this supposed to be `string | null`, maybe?
```suggestion
								{ ( annualSaving: { saving: string | null } ) =>
```

Reply:
You are right, `string | null` is the correct definition